apps/ledger-live-desktop/tests/models/Modal.ts

```diff
@@ -83,4 +103,45 @@ export class Modal {
   async chooseStakeSupportLink(stakeProvider: string) {
     await this.stakeProviderSupportLink(stakeProvider).dispatchEvent("click");
   }
+
+  async getTitleProvider() {
+    await this.titleProvider.waitFor();
+    const provider = await this.titleProvider.textContent();
+    return provider;
+  }
+
+  async continueDelegate() {
+    await this.delegateContinueButton.click();
+  }
+
+  async getSpendableBannerValue() {
+    const amountValue = await this.spendableBanner.textContent();
+    return parseInt(amountValue!.replace(/[^0-9.]/g, ""));
+  }
+
+  async toggleMaxAmount() {
+    await this.maxAmountCheckbox.click();
+  }
+
+  async getCryptoAmount() {
+    const valueAmount = await this.cryptoAmountField.inputValue();
+    return parseInt(valueAmount);
+  }
+
+  async countinueSendAmount() {
+    await this.continueAmountButton.click();
+  }
+
+  async openSearchProviderModal() {
+    await this.searchOpenButton.click();
+  }
+
+  async inputProvider(provider: string) {
+    await this.inputSearchField.fill(provider);
+  }
+
+  async selectProvider() {
```

Comment:
Totally optional, but you could make this dynamic so you can choose the provider in future:

```
 async selectProvider(providerIndex: number ) {
   await this.rowProvider.nth(providerIndex).click();
    await this.searchCloseButton.click();
  }